lib/util/emoji.ts

```diff
@@ -1,16 +1,84 @@
-import emoji from 'node-emoji';
+import is from '@sindresorhus/is';
+import mathiasBynensEmojiRegex from 'emoji-regex';
+import {
+  fromCodepointToUnicode,
+  fromHexcodeToCodepoint,
+  fromUnicodeToHexcode,
+} from 'emojibase';
+import githubShortcodes from 'emojibase-data/en/shortcodes/github.json';
+import emojibaseEmojiRegex from 'emojibase-regex/emoji';
+import SHORTCODE_REGEX from 'emojibase-regex/shortcode';
 import type { RenovateConfig } from '../config/types';
+import { regEx } from './regex';
 
 let unicodeEmoji = false;
 
+const shortCodesByHex = new Map<string, string>();
+const hexCodesByShort = new Map<string, string>();
+
+function initMappings(): void {
+  if (shortCodesByHex.size === 0 && shortCodesByHex.size === 0) {
+    for (const [hex, val] of Object.entries(githubShortcodes)) {
+      const shortcodes: string[] = is.array<string>(val) ? val : [val];
+      shortCodesByHex.set(hex, `:${shortcodes[0]}:`);
+      shortcodes.forEach((shortcode) => {
+        hexCodesByShort.set(`:${shortcode}:`, hex);
+      });
+    }
+  }
+}
+
 export function setEmojiConfig(_config: RenovateConfig): void {
+  initMappings();
   unicodeEmoji = _config.unicodeEmoji;
 }
 
+const shortcodeRegex = regEx(SHORTCODE_REGEX.source, 'g');
+
 export function emojify(text: string): string {
-  return unicodeEmoji ? emoji.emojify(text) : text;
+  return unicodeEmoji
+    ? text.replace(shortcodeRegex, (shortcode) => {
+        const hexCode = hexCodesByShort.get(shortcode);
+        return hexCode
+          ? fromCodepointToUnicode(fromHexcodeToCodepoint(hexCode))
+          : shortcode;
+      })
+    : text;
+}
+
+const emojiRegexSrc = [emojibaseEmojiRegex, mathiasBynensEmojiRegex()].map(
+  ({ source }) => source
+);
+const emojiRegex = new RegExp(`(?:${emojiRegexSrc.join('|')})`, 'g');
+const excludedModifiers = new Set([
+  '20E3',
+  '200D',
+  'FE0E',
+  'FE0F',
+  '1F3FB',
+  '1F3FC',
+  '1F3FD',
+  '1F3FE',
+  '1F3FF',
+  '1F9B0',
+  '1F9B1',
+  '1F9B2',
+  '1F9B3',
+]);
```

Comment:
`github` shortcodes take x3 (45 Kb json) less space than `cldr` (145 Kb json), and modifier stripping leaves us just about 200 most exotic symbols unavailable